### ISSUE TITLE
Rails 7 dependency - activemodel

### ISSUE THREAD

**Issue by dpaluy** [adzap/validates_timeliness#221]
Please update the Rails 7 dependency 
https://github.com/adzap/validates_timeliness/blob/v6.0.0.beta1/validates_timeliness.gemspec#L20

It should be:

`s.add_runtime_dependency("activemodel", [">= 6.0.0", "< 8"])`

**Comment by ariccio** [comment]
Looks like dependabot has already done it!
https://github.com/adzap/validates_timeliness/pull/214

Just needs to be merged. I suspect @adzap is quite a busy man! [His Linkedin page](https://www.linkedin.com/in/meehanadam) says he's director of a clinical trial research company.

I feel bad! Looks like he's one of us open source devs, doing this repo on his free time?

**Comment by foucist** [comment]
#213 will fix this (and the default_timezone warning) once it is merged

**Comment by brentgreeff** [comment]
Any update? - 

```
Bundler could not find compatible versions for gem "activemodel":
  In snapshot (Gemfile.lock):
    activemodel (= 7.0.3)
```

**Comment by YassineDM** [comment]
Same requirement here...
